app/components/plan/Workout.js

```diff
@@ -1,52 +1,80 @@
+            </div>
+            : completedWorkout.comments ?
+            <InfoItem icon="speaker_notes" label="Comments" iconColor={ textLight }
+              content={ completedWorkout.comments } /> : null
+        }
+      </div>
+      <div className="workout-banner">
+        <div className="workout-info">
+          <h2>
+            <Link to={`/plan/treatments/${treatment.id}`}>
+              {treatment.exercise.title}
+            </Link>
+          </h2>
+          <div>
+            <InfoItem icon="all_inclusive" label="Sets" iconColor={ textLight }
+              content={ treatment.sets } />
+            <InfoItem icon="layers" label="Reps" iconColor={ textLight }
+              content={ treatment.reps } />
+          </div>
+          <div className="bonus-info">
+            <InfoItem icon="alarm" label="Time" iconColor={ textLight }
+              content={ formatTime(treatment.time_per_exercise) } />
+            <InfoItem icon="fitness_center" label="Resistance" iconColor={ textLight }
+              content={ treatment.resistance } />
+          </div>
+        </div>
+        <div className="workout-button">
+        {
+          completedWorkout ?
+            <StepsActionButton disabled={true}>
+              {
+                completedWorkout.pain ?
+                  <img src={require(`../../../src/images/emojis/${completedWorkout.pain}pain.svg`)} /> :
+                  <FontIcon className={'material-icons'}>check</FontIcon>
+              }
+            </StepsActionButton> :
+            <Link to={`/plan/treatments/${treatment.id}/workout`}>
+              <StepsActionButton>
+                <FontIcon className={'material-icons'}>schedule</FontIcon>
+              </StepsActionButton>
+            </Link>
+        }
         </div>
-        <p><span>Time</span>{`: ${formatTime(treatment.time_per_exercise)}`}</p>
-        <p><span>Resistance</span>{`: ${treatment.resistance}`}</p>
       </div>
-      <p className="col-xs-4">
-        <span>Notes</span>
-        {`: ${treatment.notes ? treatment.notes : 'No notes specified'}`}
-      </p>
-      <div className="col-xs-2 workout-button">
-      {
-        completedWorkout ?
-          <StepsActionButton disabled={true}>
-            {/* TODO: Currently defaulting to pain level 3 when no input -> need real default */}
-            {
-              completedWorkout.pain ?
-                <img src={require(`../../../src/images/emojis/${completedWorkout.pain}pain.svg`)} /> :
-                <FontIcon className={'material-icons'}>check</FontIcon>
-            }
-          </StepsActionButton> :
-          <Link to={`/plan/treatments/${treatment.id}/workout`}>
-            <StepsActionButton>
-              <FontIcon className={'material-icons'}>schedule</FontIcon>
-            </StepsActionButton>
-          </Link>
-      }
+      <div className="workout-index">#{num}</div>
+      <div className="workout-pic-wrapper">
+        <img
+          className="workout-pic"
```

Comment:
would suggest also adding 'img-responsive' to className here to help control img height/width relative to its container sizing.